codescan/schema_test.go

```diff
@@ -793,6 +793,7 @@ func TestEmbeddedAllOf(t *testing.T) {
 	models := make(map[string]spec.Schema)
 	require.NoError(t, prs.Build(models))
 	schema := models["AllOfModel"]
+	require.Len(t, models, 1)
```

Comment:
This is odd I think because we do have a case for allOf where we do actually want extra models to be generated and end up as separate definitions